### ISSUE TITLE
Importing cerberus takes too long

### ISSUE THREAD

**Issue by socketpair** [pyeve/cerberus#602]
You can check your program using `python-X importtime` + https://kmichel.github.io/python-importtime-graph/

Why ? because of `pkg_resources`

![image](https://github.com/pyeve/cerberus/assets/1157412/b78e5e57-2bf0-4fe8-bdcc-f5ecbba0a9c1)

Why `pkg_resources`?

Because `__init__.py`:

Only for this:

```python
try:
    __version__ = get_distribution("Cerberus").version
except DistributionNotFound:
    __version__ = "unknown"
```

PLEASE stop using `pkg_resources`. it's PAINFULLY SLOW

**Comment by socketpair** [comment]
Is `__version__` ever required?  Is it a part of the documented API ? If not, just remove it.

**Comment by socketpair** [comment]
see #579

**Comment by funkyfuture** [comment]
solved w/ 81e2dfd6f79847e68133ce42b78dc3bb55870268
